Makefile

```
@@ -269,9 +269,17 @@ git-attributes:
 lint: .venv/requirements-timestamp
 	$(VENV_BIN)/flake8
 
+.PHONY: test-postgres
+test-postgres:
+	psql postgresql://postgres:postgres@localhost:5432 -t -c "select 'test postgres';"
+
+.PHONY: test-postgis
+test-postgres:
+	psql postgresql://postgres:postgres@localhost:5432 -t -c "select 'test postgres';"
+
 .PHONY: test-core
 test-core: .venv/requirements-timestamp
-	$(VENV_BIN)/py.test -vv $(PYTEST_OPTS) --cov-config .coveragerc --cov $(PACKAGE) --cov-report term-missing:skip-covered tests/core
+	$(VENV_BIN)/py.test -vv $(PYTEST_OPTS) --cov-config .coveragerc --cov $(PACKAGE) --cov-report term-missing:skip-covered --ignore tests/core/sources tests/core
```

```suggestion
	$(VENV_BIN)/py.test -vv $(PYTEST_OPTS) --cov-config .coveragerc --cov $(PACKAGE) --cov-report term-missing:skip-covered --ignore tests/core
```
tests/core/sources should work now

---

we can reactivate sources after merging, it will not work on this branch.